src/coreclr/src/jit/importer.cpp

```diff
@@ -16246,11 +16246,21 @@ GenTree* Compiler::impAssignMultiRegTypeToVar(GenTree* op, CORINFO_CLASS_HANDLE
 }
 #endif // FEATURE_MULTIREG_RET
 
-// do import for a return
-// returns false if inlining was aborted
-// opcode can be ret or call in the case of a tail.call
+//------------------------------------------------------------------------
+// impReturnInstruction: import a return or an explicit tail call
+//
+// Arguments:
+//     block -- call returning a struct in a registers
```

Comment:
"in a registers"? grammar seems weird.

Reply:
Yikes, copy paste artifact from the method above.

Looks like `block` is unused so I'll just remove it entirely and fix up the caller.